src/main/java/model/VehicleType.java

```diff
@@ -0,0 +1,8 @@
+package model;
+
+public enum VehicleType {
```

Comment:
Be careful about `enums` for object types. They usually lead to `OCP` violation!